tee/prototype/epochproc.go

```diff
@@ -94,6 +94,13 @@ func (e *Enclave) txRoutine(
 			if err := noInconsistentExits(&stagedTxs, exiters); err != nil {
 				log.Errorf("txRoutine: checking exiters: %v", err)
 			}
+
+			log.Infof("txRoutine: removing %d exiters", len(exiters))
+			for _, exiter := range exiters {
+				log.Infof("txRoutine: removed %s", exiter.Hash().String)
```

Comment:
Tracef

Reply:
don't you need to call the `String` method?
```suggestion
				log.Tracef("txRoutine: removed %s", exiter.Hash().String())
```